digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1942,11 +1942,11 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
       "municipalityName" -> municipalityName,
       "municipalityCode" -> municipalityCode,
       "results" ->
-        groupedResults.map{result =>
+        groupedResults.filter(row => row.privateRoadName != None || row.associationId != None).map{result =>
           Map(
             "privateRoadName" -> result.privateRoadName.getOrElse(""),
             "associationId" -> result.associationId.getOrElse(""),
-            "additionalInfo" -> result.additionalInfo.getOrElse(""),
+            "additionalInfo" -> result.additionalInfo.getOrElse(99),
```

Comment:
What is the function of this 99?

Reply:
Is for a default message in the frontend (simply saying that no additional info was provided)